### PR TITLE
[Blog] Spring Boot 2.1 + Angular 7

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

File: _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+
+{% img blog/spring-boot-2-angular-7/success-at-last.png alt:"Success!" width:"800" %}{: .center-image }
+
+You can see the full source code for the application developed in this tutorial on GitHub at [https://github.com/oktadeveloper/okta-spring-boot-2-angular-7-example](https://github.com/oktadeveloper/okta-spring-boot-2-angular-7-example).
+
+## Learn More about Spring Boot and Angular
+
+It can be tough to keep up with fast-moving frameworks like Spring Boot and Angular. This article is meant to give you a jump start on the latest releases. Angular 7 is rumored to ship with a newer, faster renderer (codenamed: [Ivy Renderer](http://ivy.angular.io/)). For specific changes, see [Angular's changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md). For Spring Boot, see its [2.1 Release Notes(https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes).
+
+This article uses [Okta's Angular SDK]((https://www.npmjs.com/package/@okta/okta-angular). To learn more about this project, or help improve it, [see its GitHub project](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-angular). I'd love to make it even easier to use!
```
Comment (jakubvul) [inline _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md]: Bad link syntax here

File: _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md [hunk 2]
```diff
@@ -0,0 +1,1082 @@
...
+
+Restart your server and celebrate when it all works! 🎉
+
+{% img blog/spring-boot-2-angular-7/success-at-last.png alt:"Success!" width:"800" %}{: .center-image }
+
+You can see the full source code for the application developed in this tutorial on GitHub at [https://github.com/oktadeveloper/okta-spring-boot-2-angular-7-example](https://github.com/oktadeveloper/okta-spring-boot-2-angular-7-example).
+
+## Learn More about Spring Boot and Angular
+
+It can be tough to keep up with fast-moving frameworks like Spring Boot and Angular. This article is meant to give you a jump start on the latest releases. Angular 7 is rumored to ship with a newer, faster renderer (codenamed: [Ivy Renderer](http://ivy.angular.io/)). For specific changes, see [Angular's changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md). For Spring Boot, see its [2.1 Release Notes(https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes).
```
Comment (jakubvul) [inline _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md]: Release notes link here is also malformed

File: _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md [hunk 3]
```diff
@@ -0,0 +1,1082 @@
...
+    );
+  }
+}
+```
+
+Now if you restart your client, you should see a login button.
+
+{% img blog/spring-boot-2-angular-7/login-button.png alt:"Login Button" width:"800" %}{: .center-image }
+
+Notice that this shows elements from the car-list component. To fix this, you can create a home component and make it the default route.
```
Comment (jakubvul) [inline _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md]: "car-list" should be code formatted here, right?

File: _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md [hunk 4]
```diff
@@ -0,0 +1,1082 @@
...
+
+```bash
+cd client
+npm install @angular/material@6.4.6 @angular/cdk@6.4.6
+```
+
+Run `npm audit fix --force` to fix outdated dependencies.
+
+```bash
+$ npm audit fix --force
```
Comment (aaronpk) [inline _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md]: Extra `$` here

File: _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md [hunk 5]
```diff
@@ -0,0 +1,1082 @@
...
+
+### Okta's Angular Support
+
+The Okta Angular SDK is a wrapper around [Okta Auth JS](https://github.com/okta/okta-auth-js), which builds on top of OIDC. More information about Okta's Angular library can be [found on npmjs.com](https://www.npmjs.com/package/@okta/okta-angular).
+
+{% img blog/spring-boot-2-angular-7/okta-angular.png alt:"Okta Angular" width:"800" %}{: .center-image }
+
+To install it, run the following command in the `client` directory:
+
+```
```
Comment (aaronpk) [inline _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md]: missing `bash` formatter

File: _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md (hunk 4 above)
Comment (aaronpk) [reply _source/_posts/2018-08-22-basic-crud-angular-7-and-spring-boot-2.md]: gotcha!

**Pull request merged**